### PR TITLE
added path parameters examples to URL BUILDING section

### PR DESCRIPTION
**Pull request by mattiaizzi**
I added the example code to show how to access to path params.

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Reference/Routes.md [hunk 1]
```diff
@@ -227,8 +227,23 @@ checked before parametric and wildcard.*
 
 ```js
 // parametric
-fastify.get('/example/:userId', (request, reply) => {})
-fastify.get('/example/:userId/:secretToken', (request, reply) => {})
+fastify.get('/example/:userId', (request, reply) => {
```
Comment (jsumners) [inline docs/Reference/Routes.md:230]: I realize they were anonymous arrow functions before your change, but please make all of these handler function full function declarations, e.g.:

```js
function (request, reply) {}
```

File: docs/Reference/Routes.md [hunk 2]
```diff
@@ -227,8 +227,23 @@ checked before parametric and wildcard.*
...
+    /**
+     * curl ${app-url}/example/12345
+     * userId === '12345'
+     */
```
Comment (jsumners) [inline docs/Reference/Routes.md:234]: Please keep jsdoc delimited comments reserved for actual jsdoc. Replace all `/** */` blocks with simple `//` blocks:

```suggestion
     // curl ${app-url}/example/12345
     // userId === '12345'
```

**Review by jsumners — CHANGES_REQUESTED** [review]

File: docs/Reference/Routes.md (hunk 1 above)
Comment (mattiaizzi) [reply docs/Reference/Routes.md]: done this

File: docs/Reference/Routes.md (hunk 2 above)
Comment (mattiaizzi) [reply docs/Reference/Routes.md]: done this

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Pull request merged**